index.html

```diff
@@ -519,10 +520,10 @@ <h3>Proofs</h3>
         </pre>
 
         <p>
-The proof example above uses the <code>JcsSignature2020</code>
-<a>proof type</a> to produce a verifiable digital proof by canonicalizing the
-input data using the JSON Canonicalization Scheme [[RFC8785]] and then
-digitally signing it using an Ed25519 elliptic curve signature.
+The proof example above suggests an exemplary <code>json-signature-2022</code>
```

Comment:
```suggestion
The proof example above suggests an example <code>json-signature-2022</code>
```

Reply:
See https://github.com/w3c/vc-data-integrity/pull/59/files#r986298535 for an alternate suggestion.